From 524e62ba6c6592320807ef94036730f135c02395 Mon Sep 17 00:00:00 2001
From: "W. Michael Petullo" <mike@flyn.org>
Date: Tue, 30 Nov 2010 02:35:46 -0600
Subject: [PATCH] Read HYPERVISOR and HYPERVISOR_ARGS from
 /etc/sysconfig/kernel and set mbkernel and mbargs Signed-off-by: W. Michael
 Petullo <mike@flyn.org>

---
 new-kernel-pkg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/new-kernel-pkg b/new-kernel-pkg
index 5e04a50..4ac1d81 100755
--- a/new-kernel-pkg
+++ b/new-kernel-pkg
@@ -89,8 +89,8 @@ moddep=""
 verbose=""
 makedefault=""
 package=""
-mbkernel=""
-mbargs=""
+mbkernel="$HYPERVISOR"
+mbargs="$HYPERVISOR_ARGS"
 adddracutargs=""
 addplymouthinitrd=""
 
-- 
1.8.3.1

